store-and-retrieve-data/spin-up-local-devnet.md

```diff
@@ -18,6 +18,20 @@ Before you begin, ensure you have the following:
 - [Rust ≥ 1.86](https://rust-lang.org/tools/install/){target=\_blank} installed
 - [PostgreSQL](https://www.postgresql.org/download/){target=\_blank} installed 
 - [Diesel CLI](https://diesel.rs/guides/getting-started#installing-diesel-cli){target=\_blank} installed for database management
+    - Install system dependencies:
+        - **Linux**:
+        ```bash
+        sudo apt update && sudo apt install build-essential
+        ```
+
+        - **macOS**:
+        ```bash
+        xcode-select --install
+        ```
+    - Then install Diesel CLI with PostgreSQL support:
```

Comment:
The way it's formatted here is a little tricky to tell if Diesel CLI is needed for both mac and linux or just mac. This could be clarified perhaps with different alignment or wording to indicate like, "Then regardless of platform you'll need to install"